src/main/java/patio/voting/domain/Voting.java

```diff
@@ -190,4 +193,22 @@ public VotingStats getStats() {
   public void setStats(VotingStats votingStats) {
     this.stats = votingStats;
   }
+
+  /**
+   * Returns if a voting is already expired or not
+   *
+   * @return
+   */
+  public Boolean getHasExpired() {
```

Comment:
I would prefer to name it just `expired` and then the getter `hasExpired`. Besides, why `Boolean` instead of `boolean` ?